### PR TITLE
javascript-operators-and-expressions-Bishan-Webeshete

### PR DESCRIPTION
**Pull request by BishanWebeshete**
Screen Shot 2023-01-30 at 12 09 55 PM

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+console.log('value ph isAcidic: ', isAcidic);
+console.log('type of isAcidic: ', typeof isAcidic);
+
+var headCount = 256;
+var isSparta = headCount === 300;
+console.log('value of isSparta: ', isSparta);
+console.log('type of isSparta: ', typeof isSparta);
+
+var motto = fullName;
+motto = motto + ' is the GOAT';
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:37]: Use the Addition assignment operator here (`+=`)

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Bishan! One small change requested below

**Comment by BishanWebeshete** [comment]
Screen Shot 2023-01-30 at 12 24 34 PM

File: javascript-operators-and-expressions/main.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+console.log('value ph isAcidic: ', isAcidic);
+console.log('type of isAcidic: ', typeof isAcidic);
+
+var headCount = 256;
+var isSparta = headCount === 300;
+console.log('value of isSparta: ', isSparta);
+console.log('type of isSparta: ', typeof isSparta);
+
+var motto = fullName;
+motto = motto += ' is the GOAT';
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js:37]: You don't need `motto =`. Line 37 would look just like this `motto += ' is the GOAT';`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change requested below

**Comment by BishanWebeshete** [comment]
Screen Shot 2023-01-30 at 1 33 54 PM

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**